### PR TITLE
Change graphql dependency version

### PR DESCRIPTION
**Pull request by crubier**
See #72

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "babel-runtime": "~5.8.3"
   },
   "peerDependencies": {
-    "graphql": "~0.4.2"
+    "graphql": "~0.5.0"
```
Comment (dschafer) [inline package.json]: Looks like we need to change line 63 as well, if I'm reading the travis run correctly?

**Comment by wincent** [comment]
I don't think we can merge this as-is because there are compatibility-breaking changes in graphql v0.5.0 (see #74).

**Comment by wincent** [comment]
I've ended up merging #74, which includes fixes for the compatibility-breaking changes as well, so going to close this one for now. Thanks once again for this, @crubier!

**Comment by crubier** [comment]
You're welcome, I didn't do much except giving you more work. Thank you for your work @wincent , I will now be able to use the latest versions in my build !

**Comment by wincent** [comment]
Note that if you do choose to use a `graphql` v0.5.0 or later you will probably have to make a couple of tweaks to your schema, matching the changes referred to in that PR. Specifically look at the first two bullet point under the "Breaking" section in [the release notes](https://github.com/graphql/graphql-js/releases/tag/v0.5.0). Sorry about that! We aim to do a better job of deprecation moving forward.

**Pull request closed without merging**